nbnovnc/static/tree.js

```diff
@@ -21,7 +21,7 @@ define(function(require) {
         var vnc_link = $('<a>')
             .attr('role', 'menuitem')
             .attr('tabindex', '-1')
-            .attr('href', base_url + 'novnc/?path=' + base_url.slice(1,) + 'novnc/websockify')
+            .attr('href', base_url + 'novnc/?host=' + window.location.host + base_url + 'novnc/&resize=remote&autoconnect=1')
```

Comment:
Have you tested this change on a jupyterhub?

Reply:
Yes. Its work !
I test it with dockerSpwaner and KubeSpawner.